exporter/debugexporter/testdata/config_verbosity.yaml

```diff
@@ -2,3 +2,6 @@ verbosity: detailed
 sampling_initial: 10
 sampling_thereafter: 50
 use_internal_logger: false
+output_paths:
```

Comment:
It would be probably better to create separate test configurations for separate tests, but it's a minor nit and can be done in a separate PR.